src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -34,7 +34,11 @@ public CommandResult execute(Model model) {
                 String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
     }
 
-    @Override
```

Comment:
Perhaps the @Override annotation should be kept, since it actually overrides the equals method of the Object superclass which it inherits? 🐱

Reply:
Yep added back thanks!